flask_common/mongo/utils.py

```diff
@@ -34,7 +34,12 @@ def iter_no_cache(query_set):
 
 
 def fetch_related(
-    objs, field_dict, cache_map=None, extra_filters={}, batch_size=100
+    objs,
+    field_dict,
+    cache_map=None,
+    extra_filters={},
+    batch_size=100,
+    filter_funcs={},
```

Comment:
I know we're not altering `filter_funcs` and `extra_filters` within this function, but should we really default to mutable types? Also, shouldn't `flake8` catch this?

Reply:
If we want to change this, I suggest we do it in a separate PR later that ensures the rules are enforced everywhere.